qiskit/pulse/instruction_schedule_map.py

```diff
@@ -52,7 +52,7 @@
 class InstructionScheduleMap:
     """Mapping from :py:class:`~qiskit.circuit.QuantumCircuit`
     :py:class:`qiskit.circuit.Instruction` names and qubits to
-    :py:class:`~qiskit.pulse.Schedule` s. In particular, the mapping is formatted as type::
+    :py:class:`~qiskit.pulse.Schedule` s. In particular, the mapping is formatted as type
```

Comment:
This was a bad change. `::` is a special symbol in Sphinx to start a code block

```suggestion
    :py:class:`~qiskit.pulse.Schedule` s. In particular, the mapping is formatted as type::
```